pkg/controller/storagecluster/controller_test.go

```diff
@@ -6269,7 +6433,7 @@ func TestHistoryCleanup(t *testing.T) {
 
 	runningPod2, err := k8scontroller.GetPodFromTemplate(&podControl.Templates[0], cluster, clusterRef)
 	require.NoError(t, err)
-	require.Equal(t, revisions.Items[0].Labels[defaultStorageClusterUniqueLabelKey],
```

Comment:
seems the expected revision was not correct, don't know how that passed. but after review, after cluster update and for new created runningPod2, it should be using the latest revision.